docker-compose.yaml

```
@@ -0,0 +1,37 @@
+services:
+  backend:
+    build: .
+    container_name: healoscope-backend
+    ports:
+      - "8080:8080"
+    environment:
+      PORT: 8080
+      MONGODB_URI: mongodb://mongo:27017/healoscope
+      JWT_SECRET: ${JWT_SECRET}
+      EMAIL_USER: ${EMAIL_USER}
+      EMAIL_PASS: ${EMAIL_PASS}
+      CLOUDINARY_NAME: ${CLOUDINARY_NAME}
+      CLOUDINARY_API_KEY: ${CLOUDINARY_API_KEY}
+      CLOUDINARY_API_SECRET: ${CLOUDINARY_API_SECRET}
+      GEMINI_API_KEY: ${GEMINI_API_KEY}
+    depends_on:
+      mongo:
+        condition: service_healthy
+    restart: unless-stopped
+
+  mongo:
+    image: mongo
+    container_name: healoscope-mongo
+    ports:
+      - "27017:27017"
+    volumes:
+      - mongo_data:/data/db
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "mongo", "--quiet", "127.0.0.1/healoscope", "--eval", "db.runCommand('ping').ok"]
```

Not working - test before you push @hirux06.


```suggestion
      test: echo 'db.runCommand("ping").ok' | mongosh localhost:27017/test --quiet
```